src/libp2p.rs

```diff
@@ -1,23 +1,10 @@
-use crate::primitives::Epoch;
```

Comment:
nitpick but I'd just leave this as `networking.rs` in the top-level

Reply:
Agreed